### ISSUE TITLE
VZ Data Quality check

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/atd-data-tech#7187]
from XA on 9/16 am:

> Good morning y'all! I was in the process of updating crashes this morning and noticed that we haven't had a new crash since 08-Sep-2021. I went onto the CRIS database and saw that there are 206 crashes that have a crash date after 08-Sep-2021. Is there an issue with the Extraction?

**Comment by mateoclarke** [comment]
![Screen Shot 2021-09-16 at 10 22 56 PM](https://user-images.githubusercontent.com/5697474/133719141-bc5f4ce2-2e01-4325-92c0-4a8304342fbd.png)

We have crashes up to 9/9 in our VZ Database. 

I did a manual query in CRIS Query Builder and I see new crashes in Travis County up to yesterday. 

![Screen Shot 2021-09-16 at 10 25 51 PM](https://user-images.githubusercontent.com/5697474/133719530-b294cd2d-92ce-4907-8556-9d88d13fe811.png)

I'm unable to login into TXDOT Extract Portal (different from QueryBuilder). Neither of my credentials are working and I won't be able to contact TXDOT support until Monday.

I checked our S3 bucket and I found more recent data in a two buckets. 
```
s3://atd-vision-zero-database/csv-extracts-sftp-staging/extract_2018_20210408151523_crash_20210915_HAYSTRAVISWILLIAMSON.csv

s3://atd-vision-zero-database/csv-extracts-sftp/extract_2018_20210408151523_crash_20210915_HAYSTRAVISWILLIAMSON.csv
```

![Screen Shot 2021-09-16 at 10 40 17 PM](https://user-images.githubusercontent.com/5697474/133721467-987bc7e8-76e7-4ba2-8285-ee93b49f55ed.png)

Records in those same buckets from yesterday looks like it only went back to 9/9.

![Screen Shot 2021-09-16 at 10 48 18 PM](https://user-images.githubusercontent.com/5697474/133721500-3338c2ab-d634-4cf5-806b-5483ff96850b.png)

So my hypothis is that nothing is broken, TXDOT just got behind and we should get newer data processed through the workflow in the next 24 hours. I'm going to see if one of the other devs is able to triage while I'm out over the weekend.
